documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -1,11 +1,19 @@
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="creating-migration-plan-2-6-3_{context}"]
-ifdef::provider[]
-= Creating a migration plan starting with a source provider
+[id="creating-migration-plan-2-8_{context}"]
+= Creating a migration plan
 
-You can create a migration plan based on a source provider, starting on the *Plans for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+You can create a migration plan by using the {ocp} web console to specify a source provider, the virtual machines (VMs) you want to migrate, and other plan details.
```

Comment:
Use the {ocp} web console to create a migration plan. Specify the source provider, the virtual machines (VMs) you want to migrate, and other plan details.

(I'm assuming the other plan details are obvious and/or optional.